QRCoder/BitmapByteQRCode.cs

```diff
@@ -70,36 +92,28 @@ public byte[] GetGraphic(int pixelsPerModule, byte[] darkColorRgb, byte[] lightC
         bmp.AddRange(new byte[] { 0x01, 0x00, 0x18, 0x00 });
         bmp.AddRange(new byte[24]);
 
+        //Group of pixels placeholder
+        var group = new byte[(int)(sideLength / pixelsPerModule) * moduleDark.Length];
```

Comment:
This allocation could be eliminated by:
- writing the first group
- storing the index of the start of the first group
- writing the remaining rows (pixelsPerModule) by simply copying the data that already exists within the array